sickbeard/name_parser/parser.py

```diff
@@ -121,7 +121,7 @@ def _parse_string(self, name):  # pylint: disable=too-many-locals, too-many-bran
 
         matches = []
         bestResult = None
-        guess = guessit.guessit(name, dict(show_type=self.show_type))
+        guess = guessit.guessit(name, dict(show_type=self.show_type, advanced=True, implicit=True))
         if self.use_guessit:
```

Comment:
@ratoaq2 : this was suggested by @labrys  ( https://thepb.in/p/GZhWNY7gY6LtV#H3-4 )

can we do that in the parser?

@p0psicles